Release/src/CMakeLists.txt

```diff
@@ -131,6 +131,13 @@ if(CPPREST_HTTP_CLIENT_IMPL STREQUAL "asio")
   target_compile_definitions(cpprest PUBLIC -DCPPREST_FORCE_HTTP_CLIENT_ASIO)
   target_sources(cpprest PRIVATE http/client/http_client_asio.cpp http/client/x509_cert_utilities.cpp)
   target_link_libraries(cpprest PUBLIC cpprestsdk_boost_internal cpprestsdk_openssl_internal)
+elseif(CPPREST_HTTP_CLIENT_IMPL STREQUAL "curl")
+  cpprest_find_boost()
```

Comment:
Hmmm do we still need boost with the curl implementation?

Reply:
It is needed for the rest of the cpprestsdk functionality. We are just replacing the http client library not the boost library.